flow/visualize/visualizer_rllib.py

```diff
@@ -160,6 +169,11 @@ def visualizer_rllib(args):
     else:
         env = gym.make(env_name)
 
+    if hasattr(env, "reroute_on_exit"):
+        env.reroute_on_exit = False
```

Comment:
Shouldn't this be True?

Reply:
Nah, you want to turn it off at test time. 'll leave a comment